apps/site/lib/site_web/controllers/schedule/timetable_controller.ex

```diff
@@ -147,25 +147,45 @@ defmodule SiteWeb.ScheduleController.TimetableController do
   """
   @spec trip_messages(Routes.Route.t(), 0 | 1) :: %{{String.t(), String.t()} => String.t()}
   def trip_messages(%Routes.Route{id: "CR-Franklin"}, 0) do
-    train = "731"
```

Comment:
Was this intentionally removed?

Reply:
Yeah. It's not in the list of trains indicated in the ticket. And it matches with the PDF schedule.